Cargo.toml

```diff
@@ -10,12 +10,14 @@ license = "MIT/Apache-2.0"
 repository = "https://github.com/filecoin-project/ff-cl-gen"
 
 [dependencies]
-ff = { version = "0.3.0", package = "fff" }
 itertools = { version = "0.8.0" }
-num-bigint = "0.2"
+pairing = "0.21.0"
 
-[dev-dependencies]
-ocl = { version = "0.19.4", package = "fil-ocl"}
-paired = { version = "0.22.0" }
-rand = "0.7"
-lazy_static = "1.2"
+[workspace]
+members = [
+  "gen-test"
+]
+
+[patch."https://github.com/filecoin-project/ff-cl-gen".ff-cl-gen]
+path = "./"
+features = ["gpu"]
```

Comment:
Is this in order to enable the `gpu` feature in `gen-test`? I guess enabling the feature there won't work?

Can a crate with `patch` section be published?

Reply:
No, this needs to go away in some form before publishing. This is a hack around the circular dependency issue